src/watchmaker/workers/salt.py

```diff
@@ -240,8 +240,16 @@ def _get_formulas_conf(self):
                 self.salt_formula_root))[1]
         ]
 
+    def _clear_copy_dir(self, src, dst, sub):
```

Comment:
I think it makes more sense to handle the `sub` outside this method, as we don't really need `src` or `dst` otherwise. Function signature would be more clear as just `def _clear_copy_dir(src, dst):` (since `self` isn't used either)

Reply:
@YakDriver, I don't see any tests :|

Reply:
@confusdcodr Challenge accepted.